phylo/searchSpace.go

```diff
@@ -0,0 +1,21 @@
+package phylo
+
+import (
+	"github.com/vertgenlab/gonomics/bed"
+)
+
+//MakeBitArrayFromSearchSpaceBed
+func MakeBitArrayFromSearchSpaceBed(searchSpaceFile string, referenceLength int, Chrom string) []bool {
+	searchSpace := bed.Read(searchSpaceFile)
+	bitArray := make([]bool, referenceLength)
```

Comment:
Does the bitArray default to all zeros when initialized or nil?

This questions springs from me wondering if the final bitArray produced from the look on lines 15-16 with a much of true and nil or true and false?